kfs-core/src/main/java/edu/arizona/kfs/fp/document/ProcurementCardDocument.java

```diff
@@ -69,5 +72,25 @@ private boolean hasReconciler() {
         }
         return retCode;
     } 
+    @Override
+    public String getDocumentTitle() {       
+        return this.buildDocumentTitle(super.getDocumentTitle());
+    }
+
+    protected String buildDocumentTitle(String title) { 
+        if(this.transactionEntries == null) {
+           return title; 
+        }
+        
+        List<ProcurementCardTransactionDetail> procurementCardTransactionDetails = new ArrayList<ProcurementCardTransactionDetail>();
+        procurementCardTransactionDetails = this.getTransactionEntries();
+        
+        String tranNumber = procurementCardTransactionDetails.get(0).getTransactionReferenceNumber().trim();
+        String vendorName = StringUtils.trimToEmpty(procurementCardTransactionDetails.get(0).getProcurementCardVendor().getVendorName());
+        String totAmount = this.getTotalDollarAmount().toString().trim();
 
-}
+        title = title + " / " + tranNumber + " / " + vendorName + " / $" + totAmount;
```

Comment:
Title Field should display the following, and in the same order: Credit card last 4, Cardholder Name, Transaction ID, Merchant Name, Transaction Amount